packages/frontend/src/page/components/opening-countdown/index.tsx

```diff
@@ -42,22 +42,22 @@ export const OpeningCountdown = ({
         <div className="flex gap-6 content-center">
             <CountdownFrame
                 label={t("label.question.time.days")}
-                timeFrame={duration.days()}
+                timeFrame={String(duration.days()).padStart(2, "0")}
```

Comment:
What's the difference between doing this and a `toString`? Btw, looks like we're using dayjs instances here, we can also use its `format` function.